minimal/glsp-client/package.json

```
@@ -0,0 +1,66 @@
+{
+  "private": true,
+  "name": "parent",
+  "version": "0.8.0",
+  "engines": {
+    "yarn": ">=1.7.0 <2.x.x",
+    "node": ">= 12.14.1"
+  },
+  "license": "(EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)",
+  "author": {
+    "name": "Eclipse GLSP"
+  },
+  "homepage": "https://www.eclipse.org/glsp/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-glsp/glsp-examples.git"
+  },
+  "bugs": "https://github.com/eclipse-glsp/glsp/issues",
+  "contributors": [
+    {
+      "name": "Eclipse GLSP Project",
+      "email": "glsp-dev@eclipse.org",
+      "url": "https://projects.eclipse.org/projects/ecd.glsp"
+    }
+  ],
+  "scripts": {
+    "prepare": "lerna run prepare",
+    "watch": "lerna run --parallel watch",
+    "build": "yarn simpleInstall && lerna run build",
+    "lint": "lerna run lint --",
+    "simpleInstall": "yarn install --ignore-scripts",
+    "rebuild:browser": "theia rebuild:browser",
+    "start": "yarn rebuild:browser && yarn --cwd minimal-browser-app start",
+    "start:external": "yarn rebuild:browser && --cwd minimal-browser-app start:external"
```

This is fixed with https://github.com/eclipse-glsp/glsp-examples/pull/89, but for the sake of completeness:
```suggestion
    "start:external": "yarn rebuild:browser && yarn --cwd minimal-browser-app start:external"
```